### PR TITLE
change cite tab

### PR DESCRIPTION
**Pull request by lizvance**

### CODE REVIEW

File: static/cli_download.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
   <div id="topnav" class='topnav'>
     <a href="index.html">HOME</a>
     <a href="calculate_score.html">CALCULATE</a>
-    <a href="cite.html">CITE</a>
+    <a href="studies.html">STUDIES</a>
```
Comment (mpage21) [inline static/cli_download.html:21]: I think the change probably needs to be made on the index.html page and the visualize.html page as well.

File: static/js/studies.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
+//ajax is asyncronous so it must call the loadStudies function on success
+function getOrderedPublicationsAndLoad() {
```
Comment (mpage21) [inline static/js/studies.js:2]: Also need to make sure the cite.html and cite.js files are deleted

**Review by mpage21 — CHANGES_REQUESTED** [review]
See my comments

File: static/cli_download.html (hunk 1 above)
Comment (lizvance) [reply static/cli_download.html:21]: Done :)

File: static/js/studies.js (hunk 2 above)
Comment (lizvance) [reply static/js/studies.js]: Done :)

**Pull request merged**